common/views/components/styled/SpacingComponent.tsx

```diff
@@ -38,6 +38,10 @@ const SpacingComponent = styled.div.attrs<{ sliceType?: string }>(props => ({
       margin-top: ${props => props.theme.spacedTextTopMargin};
     }
   }
+
+  &.slice-type-image-or-icons-and-text + &.slice-type-image-or-icons-and-text {
```

Comment:
To remind myself; the slice names will actually be separate won't they? or will we be hardcoding that one in?

Reply:
you're right – I guess we'll have to have 
```
&.slice-type-image-and-text + &.slice-type-icons-and-text,
&.slice-type-icons-and-text + &.slice-type-image-and-text,
```

Reply:
…probably as well as
```
&.slice-type-image-and-text + &.slice-type-image-and-text,
&.slice-type-icons-and-text + &.slice-type-icons-and-text,
```

to cover all the bases (i.e. for the same reason that we currently do this for text blocks)

Reply:
Yeah something like that! Not as nice and clean but reckon we'll need it.

Reply:
we _could_ do this

```
&[class*='slice-type-text-and-'] + &[class*='slice-type-text-and-']
```

but likely setting ourselves up for pain further down the line

Reply:
yeah that'd be a hard bug to find, agree not to do it.